app/api/server_version.py

```diff
@@ -1,13 +1,15 @@
 from flask import jsonify, Blueprint
+from datetime import datetime
 
+SERVER_VERSION = '2.2.3-SNAPSHOT'
 
-SERVER_VERSION = '1.0'
-
-server_version_route = Blueprint('server_version', __name__, url_prefix='/v1')
+info_route = Blueprint('info', __name__, url_prefix='/v1')
+_build = {'time': str(datetime.now()), 'version': SERVER_VERSION}
```

Comment:
time should be the time of release

Reply:
Since it'll be a little bit complex for you, ignore the time field

Reply:
@iamareebjamal sure no issues. Although if you give me some hint about the vx pipeline I can try to figure out something.

Reply:
vx pipeline?

Reply:
Simplest solution is to invoke git to find last commit time

Reply:
But we need to finalize the release today, so let it be and do it in a future release